clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookMessageHandler.ts

```diff
@@ -28,9 +40,25 @@ export async function searchCopybook(
   copybookName: string,
   dialectType: string,
   storagePath: string,
+  e4eDownloader?: CopybookDownloaderForE4E,
+  dsnDownloader?: CopybookDownloaderForDsn,
+  ussDownloader?: CopybookDownloaderForUss,
 ): Promise<Uri | undefined> {
   let result: Uri | undefined;
 
+  const procGroupResult = await searchCopybookinProcessorGroups(
+    documentUri,
+    copybookName,
+    dialectType,
+    storagePath,
+    e4eDownloader,
+    dsnDownloader,
+    ussDownloader,
+  );
+  if (procGroupResult && typeof procGroupResult != "boolean")
+    return procGroupResult;
+  else if (procGroupResult) return;
```

Comment:
Remove the new parameters and move the rest of the block before the call to `searchCopybook` in `resolveCopybookHandler`.